collector/lambdacomponents/go.mod

```diff
@@ -1,15 +1,72 @@
 module github.com/open-telemetry/opentelemetry-lambda/collector/lambdacomponents
 
-go 1.17
+go 1.18
 
 require (
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.56.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/processor/attributesprocessor v0.56.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor v0.56.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/processor/probabilisticsamplerprocessor v0.56.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor v0.56.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanprocessor v0.56.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/sigv4authextension v0.56.0
```

Comment:
This PR needs to be rebased to include sigv4auth. missing this is causing the failure in PR builds